### PR TITLE
New version: EliWeitzman.ETT version 1.2.1

### PR DESCRIPTION
**Pull request by eliweitzman**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/136790)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
>  Exception during executable launch operation System.AggregateException: One or more errors occurred. (An error occurred trying to start process 'C:\Program Files\Eli's Enterprise Tech Toolkit\ETT.exe' with working directory 'D:\TOOLS'. The specified executable is not a valid application for this OS platform.)  ---> System.ComponentModel.Win32Exception (216): An error occurred trying to start process 'C:\Program Files\Eli's Enterprise Tech Toolkit\ETT.exe' with working directory 'D:\TOOLS'. The specified executable is not a valid application for this OS platform.    --- End of inner exception stack trace ---    at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)

(Automated response - build 669.)

**Comment by eliweitzman** [comment]
This might be a defender-related error. It seems since the beginning of January there's been constant false positives. I'm submitting a false positive report to MS Security Intelligence in the meantime.

**Comment by stephengillie** [comment]
> This might be a defender-related error. It seems since the beginning of January there's been constant false positives. I'm submitting a false positive report to MS Security Intelligence in the meantime.

Defender-type failures have different text. I believe this package passed the Defender scans, but failed for another reason, such as being a 64-bit package running on a 32-bit OS, such as the ones used by our `x86` validations. Is this package 64-bit? 

> An error occurred trying to start process 'C:\Program Files\Eli's Enterprise Tech Toolkit\ETT.exe' with working directory 'D:\TOOLS'. The specified executable is not a valid application for this OS platform.

File: manifests/e/EliWeitzman/ETT/1.2.1/EliWeitzman.ETT.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: EliWeitzman.ETT
+PackageVersion: 1.2.1
+InstallerType: inno
+InstallerSwitches:
+  Silent: /SILENT /ALLUSERS
+  SilentWithProgress: /VERYSILENT /ALLUSERS
+Installers:
+- Architecture: x86
```
Comment (stephengillie) [inline manifests/e/EliWeitzman/ETT/1.2.1/EliWeitzman.ETT.installer.yaml:11]:
```suggestion
- Architecture: x64
```

**Comment by eliweitzman** [comment]
That makes sense! For some reason I was running into trouble with wingetcreate so provides better clarity. I believe the application is compiled for x64, but for some reason was getting auto-recognized as x86.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**